cloudbuild.yaml

```diff
@@ -3,7 +3,7 @@
 timeout: 2700s
 options:
   substitution_option: ALLOW_LOOSE
-  machineType: 'E2_HIGHCPU_8'
+  machineType: 'E2_HIGHCPU_32'
```

Comment:
Was the timeout maybe due to this machine type increase?

Maybe it was not able to get scheduled?

Do the other changes make sense without this line (+ setting `8` in line 15?

Reply:
yes that seems most likely and would also explain why the timeout did not occur in my environment.  the Cloudbuild queue time is visible via the GCP console but is not clear from our current logging. Having the timestamps would help us confirm that theory, but given several runs timed out I am not sure we could take any action to improve this situation anyway.

> Do the other changes make sense without this line (+ setting 8 in line 15?
Yes they do and based on my previous testing should give us ~10 mins improvement in job time

Reply:
wanted to touch back on here @chrischdi -- what do you think is the best path forward?  ill open a new PR leaving the machine type as `E2_HIGHCPU_8` and pass in `-j 8` ?

Reply:
I think that's a viable path forward 👍

Reply:
Nice catch!